TextAnalysis/src/langanal/word/processing/WordProcessing.java

```diff
@@ -0,0 +1,239 @@
+package langanal.word.processing;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+import langanal.word.base.Word;
+import langanal.word.base.WordInfo;
+
+public class WordProcessing {
+	//Weights
+	public static float definitionWeight = .5f;
+	public static float POSWeight = 0.2f;
+	public static float synonymWeight = 1;
+	public static float antonymWeight = 1;
+	
+	public static boolean debugging = false;
+
+	private static String[] commonWords = 
+			new String[]{
+					"the", "be", "to", "of", "and", "a", "in","that", "have", "i","it","for","not","not","on","with",
+					"or", "is"
+	};
+
+	/**
+	 * Compares, processes, and returns the relevancy of the two words
+	 * 
+	 * @param word1
+	 * @param word2
+	 * @return The % relevancy
+	 */
+	public static float compareWords(String word1, String word2){
+		float relevancy = 0;	//in %
+
+		LinkedList<Word> allWord1 = new LinkedList<Word>();
```

Comment:
Would an array list be better? There don't seem to be a ton of writes, but there are a lot of reads.